pandas/util/validators.py

```diff
@@ -0,0 +1,48 @@
+"""
+Module that contains many useful utilities
+for validating data or function arguments
+"""
+
+
+def validate_args(args, min_length=0, max_length=None):
+    """
+    Checks whether the length of the `*args` argument passed into a function
+    has at least `min_length` arguments. If `max_length` is an integer, checks
+    whether `*args` has at most `max_length` arguments inclusive. Raises a
+    ValueError if any of the conditions are False.
+    """
```

Comment:
can you add a test module to (in ``pandas.util``) to validate these validators, just some simple tests that exercise each path.